e2e/support/cypress.js

```diff
@@ -114,14 +114,29 @@ Cypress.on("window:load", window => {
   };
 });
 
-// cypress-terminal-report
 if (isCI) {
+  // cypress-terminal-report
   afterEach(() => {
     cy.wait(50, { log: false }).then(() =>
       cy.addTestContext(Cypress.TerminalReport.getLogs("txt")),
     );
   });
 
+  // Fast failure notifications
+  afterEach(() => {
+    const testState = Cypress.mocha.getRunner().suite.ctx.currentTest.state;
+    const testInfo = Cypress.mocha.getRunner().suite.ctx.currentTest;
```

Comment:
Nit: but couldn't we avoid repeating the whole `Cypress.mocha.getRunner().suite.ctx.currentTest` by getting the `state` out of the `testInfo`?

Reply:
https://github.com/metabase/metabase/pull/51605/commits/8902d4a394d287fe0c2ecd7ce98fb879c059475b